app/clients/sms/aws_pinpoint.py

```diff
@@ -41,16 +41,25 @@ def send_sms(self, to, content, reference, multi=True, sender=None, template_id=
             opted_out = False
             to = phonenumbers.format_number(match.number, phonenumbers.PhoneNumberFormat.E164)
             destinationNumber = to
-
+            if phonenumbers.region_code_for_number(match.number) != "CA":
```

Comment:
Let's add comment on top of the condition on why we need to do that please. 🙏 i.e. we want to use custom sender for international numbers, except Canada because it is not supported.

Reply:
Refactored slightly and added a comment:
```
# For international numbers we send with an AWS number for the corresponding country, using our default sender id.
# Note that Canada does not currently support sender ids.
if phonenumbers.region_code_for_number(match.number) != "CA":
    response = self._client.send_text_message(
        DestinationPhoneNumber=destinationNumber,
        MessageBody=content,
        MessageType=messageType,
        ConfigurationSetName=self.current_app.config["AWS_PINPOINT_CONFIGURATION_SET_NAME"],
    )
else:
    response = self._client.send_text_message(
        DestinationPhoneNumber=destinationNumber,
        OriginationIdentity=pool_id,
        MessageBody=content,
        MessageType=messageType,
        ConfigurationSetName=self.current_app.config["AWS_PINPOINT_CONFIGURATION_SET_NAME"],
    )
```